bindings/node.js/test/kzg.test.ts

```diff
@@ -81,6 +81,35 @@ function bytesEqual(a: Uint8Array | Buffer, b: Uint8Array | Buffer): boolean {
   return true;
 }
 
+function getValidTest(testDir: string): any {
```

Comment:
jsdoc: `Helper function to return first valid test in testDir`

Reply:
Updated.